flink_application.go

```diff
@@ -0,0 +1,152 @@
+package aiven
+
+type (
+	// FlinkApplicationHandler is the client which interacts with the Flink Application.
+	FlinkApplicationHandler struct {
+		client *Client
+	}
+
+	// GenericFlinkApplicationResponse is the generic response for Flink Application requests.
+	// GET https://api.aiven.io/v1/project/{project}/service/{service_name}/flink/application
+	GenericFlinkApplicationResponse struct {
+		APIResponse
+		Nameable
+		Identifiable
+		CreationMarkable
+		UpdatesMarkable
+	}
+
+	// DetailedFlinkApplicationResponse is the detailed response for Flink Application requests.
+	// GET /project/{project}/service/{service_name}/flink/application/{application_id}
+	// POST /project/{project}/service/{service_name}/flink/application
+	// PUT /project/{project}/service/{service_name}/flink/application/{application_id}
+	// DELETE /project/{project}/service/{service_name}/flink/application/{application_id}
+	DetailedFlinkApplicationResponse struct {
+		GenericFlinkApplicationResponse
+
+		ApplicationVersions []interface{} `json:"application_versions"`
+		CurrentDeployment   interface{}   `json:"current_deployment"`
+	}
+
+	// CreateFlinkApplicationRequest is the request to create a Flink Application.
+	// POST /project/{project}/service/{service_name}/flink/application
+	CreateFlinkApplicationRequest struct {
+		Nameable
+
+		ApplicationVersion interface{} `json:"application_version,omitempty"`
+	}
+
+	// UpdateFlinkApplicationRequest is the request to update a Flink Application.
+	// PUT /project/{project}/service/{service_name}/flink/application/{application_id}
+	UpdateFlinkApplicationRequest struct {
+		Nameable
+	}
+
+	// FlinkApplicationListResponse is the response for listing Flink Applications.
+	// GET /project/{project}/service/{service_name}/flink/application
+	FlinkApplicationListResponse struct {
+		APIResponse
+
+		Applications []GenericFlinkApplicationResponse `json:"applications"`
+	}
+)
+
+// Get is the method to get a Flink Application.
+func (h *FlinkApplicationHandler) Get(
+	project string,
+	service string,
+	applicationID string,
+) (*DetailedFlinkApplicationResponse, error) {
+	path := buildPath("project", project, "service", service, "flink", "application", applicationID)
+
+	bts, err := h.client.doGetRequest(path, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	var r DetailedFlinkApplicationResponse
+
```

Comment:
I this these new line break is no necessary here between two connected statements